Analysis/DataModel/include/AnalysisDataModel/HFSecondaryVertex.h

```diff
@@ -299,6 +307,85 @@ DECLARE_SOA_TABLE(HfCandProng2MCRec, "AOD", "HFCANDP2MCREC",
 DECLARE_SOA_TABLE(HfCandProng2MCGen, "AOD", "HFCANDP2MCGEN",
                   hf_cand_prong2::FlagMCMatchGen);
 
+// cascade decay candidate table
+
+namespace hf_cand_casc
+{
+DECLARE_SOA_EXPRESSION_COLUMN(Px, px, float, 1.f * aod::hf_cand::pxProng0 + 1.f * aod::hf_cand::pxProng1);
+DECLARE_SOA_EXPRESSION_COLUMN(Py, py, float, 1.f * aod::hf_cand::pyProng0 + 1.f * aod::hf_cand::pyProng1);
+DECLARE_SOA_EXPRESSION_COLUMN(Pz, pz, float, 1.f * aod::hf_cand::pzProng0 + 1.f * aod::hf_cand::pzProng1);
+DECLARE_SOA_DYNAMIC_COLUMN(M, m, [](float px0, float py0, float pz0, float px1, float py1, float pz1, const array<double, 2>& m) { return RecoDecay::M(array{array{px0, py0, pz0}, array{px1, py1, pz1}}, m); });
+
+template <typename T>
+auto InvMassLc(const T& candidate)
+{
+  return candidate.m(array{RecoDecay::getMassPDG(kK0Short), RecoDecay::getMassPDG(kProton)}); // first daughter is K0s
+}
+
+template <typename T>
+auto InvMassGamma(const T& candidate)
+{
+  return candidate.m(array{RecoDecay::getMassPDG(kElectron), RecoDecay::getMassPDG(kElectron)});
+}
+
+} // namespace hf_cand_casc
+
+DECLARE_SOA_TABLE(HfCandCascBase, "AOD", "HFCANDCASCBASE",
+                  // general columns
+                  HFCAND_COLUMNS,
+                  // cascade specific columns
+                  hf_cand::PxProng0, hf_cand::PyProng0, hf_cand::PzProng0,
+                  hf_cand::PxProng1, hf_cand::PyProng1, hf_cand::PzProng1,
+                  hf_cand::ImpactParameter0, hf_cand::ImpactParameter1,
+                  hf_cand::ErrorImpactParameter0, hf_cand::ErrorImpactParameter1,
+                  hf_track_index::IndexV0Id, // V0 index
+                  hf_track_index::Index0Id,
```

Comment:
Shouldn't the bachelor have `Index1Id` if everywhere the V0 goes first and the bachelor second?

Reply:
The V0 has index IndexV0Id, the bachelor is the first track of the candidate, so Index0Id. I would leave it as it is.